src/packet/packetlist.js

```diff
@@ -69,6 +72,12 @@ class PacketList extends Array {
           await writer.ready;
           const done = await readPackets(readable, async parsed => {
             try {
+              if (parsed.tag === MarkerPacket.tag || parsed.tag === TrustPacket.tag) {
```

Comment:
By importing both classes here, we include their implementations in every build of openpgp.js - even though they will essentially never be used. While they're admittedly very small, I think for this particular case it might be worth it to hardcode the tags here, with comments; then these classes can almost always be dead code eliminated.

Reply:
I thought it would be smart enough to just keep the tag, since it's static 😞